app/models/agents/webhook_agent.rb

```diff
@@ -80,15 +86,20 @@ def receive_web_request(params, method, format)
           error "Verification failed: #{e.message}"
           return ["Not Authorized", 401]
         end
-
+          
         JSON.parse(response.body)['success'] or
           return ["Not Authorized", 401]
       end
+        
 
       [payload_for(params)].flatten.each do |payload|
+        if interpolated['header_key'].presence 
```

Comment:
`presence` also returns the value, since we don't need it here `present?` is enough